cli_reference/developer_cli_odo/odo-cli-reference.adoc

```diff
@@ -1,3 +1,4 @@
+////
```

Comment:
Why commend this out instead of removing it together with all 
`modules/developer-cli-odo-*` files?

Reply:
I just commented it out to ensure we have legacy content in case we decide to develop odo documentation in future. If you think otherwise, please let me know. Thank you!

Reply:
It is all tracked in git. But it is really up to you and your team how you usually do this.

Reply:
Cc @pmkovar @rkratky

Reply:
Don't have a strong opinion on this but since it is indeed in git, we can as well just remove it completely.

Reply:
Since we have almost all approvals from the stakeholders, I believe we can keep what we have as is.